src/cargo/core/profiles.rs

```diff
@@ -666,6 +669,8 @@ pub struct Profile {
     pub incremental: bool,
     pub panic: PanicStrategy,
     pub strip: Strip,
+    // Note that `rustflags` is used for the cargo-feature `profile_rustflags`
+    pub rustflags: Vec<InternedString>,
```

Comment:
Can this be excluded until it is stable?  I think adding `#[serde(skip_serializing_if = "Vec::is_empty")]` should be sufficient (with a comment to make it clear it should be removed when stabilized).